src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -129,19 +129,38 @@
   {:render
    (fn [{:keys [state refs]}]
      [:div {}
+      ;; Begin overlay
+      [:div {:style {:visibility (if-not (:overlay-shown? @state) "hidden")
+                     :position "absolute"
+                     :top "0px" :left "0px" :width "100%" :height "100%"
```

Comment:
Items like `:top "0px"` can be shortened to `:top 0`. Most pixel values can be expressed this way. It's just a documented feature of React. I suggest we follow this to ensure our code is as compact as possible (which increases the amount of context that is visible on a single screen).